frontend/src/js/components/Uploads/UploadTarget.ts

```diff
@@ -21,18 +20,12 @@ export function getDefaultCollection(username: string, collections: Collection[]
     });
 }
 
-async function createUploadIngestion(collection: Collection): Promise<string> {
-    const ingestionName = `${new Date().toISOString()}`;
-    const newIngestion = await createNewIngestion(collection.uri, `Upload ${ingestionName}`);
```

Comment:
should we remove `uploadFileToIngestion` and `createNewIngestion` as they're no longer used?